pkg/Microsoft.Windows.Compatibility/tests/publishcompatibilityassets.ps1

```diff
@@ -0,0 +1,70 @@
+$repoRoot = ((get-item $PSScriptRoot).parent.parent.parent.FullName);
+$winRID = "win-x64";
+$dotnetPath = -join($repoRoot, "\Tools\dotnetcli\dotnet.exe")
+$csprojPath = -join($PSScriptRoot, "\", (Get-ChildItem $PSScriptRoot"\*.csproj" | Select-Object -ExpandProperty Name))
+$packagesCachePath = -join($repoRoot, "\packages")
+$localPackageSourcePath = -join($repoRoot, "\bin\packages\Debug\")
+$packageName = "Microsoft.Windows.Compatibility"
+
+function _pathExists($path)
```

Comment:
nit: why do you think `_pathExists path` is easier then `Test-Path path`

Reply:
I will fix it up, when I wrote the function I thought it was going to make it better but at the end it is the same.